packages/core/src/lib/normalize.ts

```diff
@@ -7,27 +7,27 @@ export async function normalizeAuditOutputs(
   const gitRoot = await getGitRoot();
 
   return audits.map(audit => {
-    if (
```

Comment:
As we have table now, I move the early exit to the issues directly

Reply:
I don't understand why you need to change anything, though. The table doesn't need to be normalized, so how is it related? Previously, the audit wouldn't be cloned at all if it didn't contain issues with a source path. After your change, all audits which have details will get cloned even though there's nothing to normalize.

Reply:
Oh, mh.. Will refactor that back. Thx for pointing that out.

Reply:
Hope it is better now.